src/coreclr/gc/gcinterface.h

```diff
@@ -570,6 +570,19 @@ enum class GCConfigurationType
 
 using ConfigurationValueFunc = void (*)(void* context, void* name, void* publicKey, GCConfigurationType type, int64_t data);
 
+enum EnableNoGCRegionCallbackStatus
```

Comment:
can you please move these next to where the current no_gc enums are defined (enum start_no_gc_region_status and etc)?